### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by jeff47** [yermak/AudioBookConverter#315] [Ideas]
A few suggestions (based on the linux version):

1.) Please fix the temp file issue, so that it deletes temp files when it's done processing each job.  As it currently stands, during the processing of my library, I filled up my temp drive completely and had to empty it manually.
2.) When starting a conversion, the file extension isn't added automatically.
3.) The "series" tag seems to get filled in automatically, and usually inappropriately.  I suggest it be set to empty until someone fills it in.

**Comment by yermak** [comment]
@jeff47,
Thanks for raising this.
1) This is on top of my priority list and will be fixed next release.
2) I will retest it on linux and see.
3) Series tag is filled from "album" tag of mp3 file and title tag is filled from "title", quite often book producers put there "book name" into "album" and quite often there is chapter name in title tag. There is no good/easy way of fixing this. I'm trying to stick with naming conventions which a used by Apple in iTunes as it's only one consistent way I found (and they sizeable market share). Sorry about this inconvinience.

**Reply by jeff47** [reply]
Understood.  Tags are a can of worms for sure - the only consistency is that there is no consistency.

Thanks - the tool works beautifully overall and really helped me get my collection organized.

**Comment by trymeouteh** [comment]
Please release this app as a Flatpak or AppImage!
